pinot-spi/src/test/resources/segment-metadata-without-version-header.properties

```diff
@@ -0,0 +1,123 @@
+segment.padding.character = \u0000
+segment.name = fineFoodReviews_OFFLINE_0
+segment.table.name = fineFoodReviews
+segment.dimension.column.names = ProductId,Score,Summary,Text,UserId,combined,embedding,n_tokens
+segment.total.docs = 1000
+custom.input.data.file.uri = ~/examples/batch/fineFoodReviews/rawdata/fine_food_reviews_with_embeddings_1k.parquet.gzip
+column.ProductId.cardinality = 868
+column.ProductId.totalDocs = 1000
+column.ProductId.dataType = STRING
+column.ProductId.bitsPerElement = 10
+column.ProductId.lengthOfEachEntry = 10
+column.ProductId.columnType = DIMENSION
+column.ProductId.isSorted = false
+column.ProductId.hasDictionary = true
+column.ProductId.isSingleValues = true
+column.ProductId.maxNumberOfMultiValues = 0
+column.ProductId.totalNumberOfEntries = 1000
+column.ProductId.isAutoGenerated = false
+column.ProductId.minValue = 7310172001
```

Comment:
Can we add some escaped values in this file? We want to ensure the escaped values can be properly read. Same for the v2 config